test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+const axios = require('axios');
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+
+chai.use(chaiSubset);
+/* Instances from the axios class that has headers attribute
+ to do the authentication */
+const object = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+
+describe('Github Api Test 5', () => {
+  describe('testing DELETE method', async () => {
+    /* test that creates a new gist with post method and checks the
+    response status code, the description, the visibility and the files */
+    it('Creating gist test', async () => {
+      const gist = await object.post('https://api.github.com/gists', { description: 'this is a example about promises', public: true, files: { 'README.md': { content: 'this is a example about promises' } } });
```

Comment:
please declare the object (second param that you are passing to the `post` method) as a global const variable, since it can be reused in the `containSubset` expect and in the other tests